lib/ansible/modules/system/aix_inittab.py

```diff
@@ -0,0 +1,261 @@
+    mkitab or rmitab commands.
+  - tested at AIX 7.1.
+requirements: [ 'itertools']
+'''
+
+EXAMPLES = '''
+# Add service startmyservice to the inittab, directly after service existingservice.
+- name: Add startmyservice to inittab
+  aix_inittab:
+    name: startmyservice
+    runlevel: 4
+    processaction: once
+    command: "echo hello"
+    follow: existingservice
+    action: install
+  become: yes
+
+# Change inittab enrty startmyservice to runlevel "2" and processactio "wait".
+- name: Change startmyservice to inittab
+  aix_inittab:
+    name: startmyservice
+    runlevel: 2
+    processaction: wait
+    command: "echo hello"
+    action: change
+  become: yes
+
+# Remove inittab entry startmyservice.
+- name: remove startmyservice from inittab
+  aix_inittab:
+    name: startmyservice
+    runlevel: 2
+    processaction: wait
+    command: "echo hello"
+    action: remove
+  become: yes
+'''
+
+RETURN = '''
+# description: The result is deliverd in an dictionary.
+- return:
+  changed: true
+  msg: []
+  name: "startmyservice"
+  status: "changed inittab entry startmyservice"
+  warnings: []
+'''
+
+# Import necessary libraries
+import itertools
+
+try:
+    import json
+except ImportError:
+    import simplejson as json
+
+from ansible.module_utils.basic import AnsibleModule
+
+# end import modules
+# start defining the functions
+
+
+def check_current_entry(module):
+    # Check if entry exists, if not return False in exists in return dict,
+    # if true return True and the entry in return dict
+    existsdict = { 'exist' : False }
+    (rc, out, err) = module.run_command(['lsitab', module.params['name']])
+    if rc == 0:
+        keys = ('name', 'runlevel', 'processaction', 'command')
+        values = out.split(":")
+        values = map(lambda s: s.strip(), values)  # strip non readable characters as \n
+        existsdict = dict(itertools.izip(keys,values))
+        existsdict.update({ 'exist' : True })
+    return existsdict
+
+
+def main():
+    # initialize
+    module = AnsibleModule(
+        argument_spec = dict(
+            name = dict( required=True, type='str', aliases=['service']),
+            runlevel = dict( required=True, type='str'),
+            processaction = dict( choices=[
+                                  'respawn',
+                                  'wait',
+                                  'once',
+                                  'boot',
+                                  'bootwait',
+                                  'powerfail',
+                                  'powerwait',
+                                  'off',
+                                  'hold',
+                                  'ondemand',
+                                  'initdefault',
+                                  'sysinit'
+                                  ], type='str'),
+            command = dict( required=True, type='str' ),
+            follow = dict( type='str' ),
+            action = dict( choices=[
```

Comment:
I will change this in the module: it will be state with choice: pressent or absent